Formula/e/exim.rb

```diff
@@ -1,8 +1,8 @@
 class Exim < Formula
   desc "Complete replacement for sendmail"
   homepage "https://exim.org"
-  url "https://ftp.exim.org/pub/exim/exim4/exim-4.96.2.tar.xz"
```

Comment:
we probably need a mirror for it, the old artifact is gone when upstream makes a new release

Reply:
Looks like old tarballs get moved to `old/` directory, e.g. https://ftp.exim.org/pub/exim/exim4/old/exim-4.96.2.tar.xz

Reply:
good catch!